### PR TITLE
LOG-1108: ClusterName for CloudWatch feature

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### Description
CLO will try to autodetect cluster name for the CloudWatch log group prefix

/cc @jcantrill 
/assign @jcantrill @vimalk78 

### Links
- JIRA: https://issues.redhat.com/browse/LOG-1108

### CODE REVIEW

File: pkg/apis/logging/v1/output_types.go [hunk 1]
```diff
@@ -33,7 +33,6 @@ type Cloudwatch struct {
 	Region string `json:"region,omitempty"`
 
 	//GroupBy defines the strategy for grouping logstreams
-	//+kubebuilder:validation:Enum:=logType;namespaceName;namespaceUUID
```
Comment (jcantrill) [inline pkg/apis/logging/v1/output_types.go:36]: Lost enum here in the merge

Comment (igor-karpukhin) [reply pkg/apis/logging/v1/output_types.go:36]: Fixed

File: manifests/5.1.beta1/logging.openshift.io_clusterlogforwarders_crd.yaml [hunk 2]
```diff
@@ -96,10 +96,6 @@ spec:
                       properties:
                         groupBy:
                           description: GroupBy defines the strategy for grouping logstreams
-                          enum:
```
Comment (jcantrill) [inline manifests/5.1.beta1/logging.openshift.io_clusterlogforwarders_crd.yaml:99]: looks like the validations are still missing from the CRD.  `make check` maybe?

**Review by jcantrill — COMMENTED** [review]
/approve
/hold

File: manifests/5.1.beta1/logging.openshift.io_clusterlogforwarders_crd.yaml (hunk 2 above)
Comment (igor-karpukhin) [reply manifests/5.1.beta1/logging.openshift.io_clusterlogforwarders_crd.yaml]: Sorry. Fixed.

**Pull request merged**